app/components/kb/stakeholder-tags.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+    <div class="mt-4" v-if="Tags">
+        <div class="loading" v-if="loading"><i class="fa fa-cog fa-spin fa-lg" ></i> {{ $t("loading") }}...</div>
+        <h4>{{ $t("stakeholders") }}</h4>
```

Comment:
use slot for generic component so each implementation can add its own title
```suggestion
       <slot name="title">
            <h4>
            {{ $t("stakeholders") }} 
            </h4>
        </slot>
```